inspectit-oce-demo/prometheus/prometheus.yml

```diff
@@ -1,6 +1,6 @@
 # my global config
 global:
-  scrape_interval:     15s
+  scrape_interval: 1s
```

Comment:
We have configured inspectit.metrics.frequency by default with 15s. If you want system metrics to be updated at a higher rate you should configure this to a lower value for each service.